pkg/ingester/user_tsdb.go

```diff
@@ -306,16 +311,17 @@ func (u *userTSDB) PreCreation(metric labels.Labels) error {
 	return nil
 }
 
-// getSeriesAndShardsForSeriesLimit returns current number of series and shard size that should be used for computing
+// getSeriesAndMinForSeriesLimit returns current number of series and minimum local limit that should be used for computing
 // series limit.
-func (u *userTSDB) getSeriesAndShardsForSeriesLimit() (int, int) {
+func (u *userTSDB) getSeriesAndMinForSeriesLimit() (int, int) {
```

Comment:
[nit] WDYT?

```suggestion
func (u *userTSDB) getSeriesCountAndMinLocalLimit() (int, int) {
```

Reply:
renamed in 5e79268c2ca3ab4227e0ff563420f507342c5dd0